lib/full_dqt/v3/client.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "net/http"
+
+# NOTE: Use this API client for looking up information on a participant once
+# we have validated their information using the V1 API.
+#
+# DO NOT USE THIS CLIENT FOR VALIDATION
+#
+# (in theory at some point in the future we could use this for validation,
+# but using the V1 API keeps the id matching logic on the API side whereas
+# using the V3 API we would need to match and validate everything our side)
+#
+module FullDQT
+  module V3
+    class Client < V1::Client
```

Comment:
Is the same api key valid for both versions of the API?

Reply:
Do we have a sandbox V3 environment of this API like those V1 we use in our review and dev environments?

Reply:
Yes and yes, it works against the same test data. I run queries in prod and dev and it all works fine with our current keys

Reply:
Great! Thanks